std/traits.d

```diff
@@ -3279,19 +3280,7 @@ alias Identity(alias A) = A;
    Yields $(D true) if and only if $(D T) is an aggregate that defines
    a symbol called $(D name).
  */
-template hasMember(T, string name)
```

Comment:
Ehm Maybe we should do the other way around - introducesMember be what hasMember template does. It's surely confusing.